com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -167,14 +167,43 @@ For more information, including the full list of light properties, see the [Ligh
 
 HDRP supports cubemap and planar GPU Reflection Probes to help you produce realistic reflections in your Scene in real time. For more information, see the [introduction to Reflection Probes](Reflection-Probes-Intro.md)
 
+### Planar reflection probe filtering
+
+![](Images/PlanarReflectionFiltering-Feature.png)
+
+Planar reflection probe filtering combines the result of planar reflection and surfaces smoothness. HDRP includes accurate filtering for planar reflection that is close to being physically-based.
+
+### Screen-space global illumination
+
+![](Images/HDRPFeatures-SSGI.png)
```

Comment:
move this one just after ### Screen-space reflection and refraction

Reply:
Remove 

"as a fallback for ray-traced global illumination (RTGI)". It is not appropriate here.

Reply:
and thus removed 
Since this feature is a fallback for RTGI, for more information, see [Ray-traced Global Illumination](Ray-Traced-Global-Illumination.md).